src/css/nav.css

```
@@ -158,17 +316,39 @@
-  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
-  border: 1px solid #ccc;
-  color: var(--body-font-color);
+  border-radius: 12px;
+  padding: 8px;
+  box-shadow: 0 8px 24px rgba(0, 0, 0, 0.32);
+  border: 1px solid rgba(255, 255, 255, 0.08);
+  color: #e8eef6;
+  animation: versionMenuIn 0.15s ease-out;
+}
+
+@keyframes versionMenuIn {
```

Stylelint (17.11.0)

[error] Expected keyframe name "versionMenuIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)